lib/helpers/files.js

```diff
@@ -121,6 +126,16 @@ function getModuleName(cwd) {
 		});
 }
 
+function getModuleBrands(cwd) {
+	return getOrigamiJson(cwd)
+		.then(origamiJson => {
+			if (origamiJson && origamiJson.brands) {
```

Comment:
Just to be safe, should we change to this?
```suggestion
			if (origamiJson && origamiJson.brands && Array.isArray(origamiJson.brands)) {
```